packages/apollo-gateway/src/loadServicesFromStorage.ts

```diff
@@ -0,0 +1,139 @@
+import { CachedFetcher } from './cachedFetcher';
+import { ServiceDefinition } from '@apollo/federation';
+import { parse } from 'graphql';
+import createSHA from './utilities/createSHA';
+
+export interface LinkFileResult {
+  configPath: string;
+  formatVersion: number;
+}
+
+export interface ImplementingService {
```

Comment:
Same questions about needing to `export`.  I didn't see where these were needed by the consumers right now, but I didn't dig much, so preference to wait until necessary to `export`, when possible?  (Most pressing thought here was that I believe this is the first time we're exposing `graphID` and elsewhere in here we seem to use `graphId` — which I personally prefer, but I assume this matches up with some JSON artifacts in GCS?)

Reply:
Removed the export.